tests/test_client.py

```diff
@@ -19,7 +19,7 @@ def api_url(path):
     return urljoin(TEST_API_URL, path)
 
 
-def add_mock_rasters_list_response():
+def add_mock_rasters_list_response(folder_id=''):
```

Comment:
I'd consider doing an extra `add_mock_rasters_in_folder_list_response` function to keep the level of branching a bit lower.

Reply:
Checked